mediafile.py

```diff
@@ -1731,8 +1736,9 @@ def update(self, dict):
     )
     label = MediaField(
         MP3StorageStyle('TPUB'),
-        MP4StorageStyle('----:com.apple.iTunes:Label'),
+        MP4StorageStyle('----:com.apple.iTunes:LABEL'),
         MP4StorageStyle('----:com.apple.iTunes:publisher'),
+        MP4StorageStyle('----:com.apple.iTunes:Label', read_only=True),
```

Comment:
Would you mind terribly if we did the “label” thing in a separate pull request from this basic functionality?